addons/portal/models/portal_mixin.py

```diff
@@ -12,6 +12,14 @@ class PortalMixin(models.AbstractModel):
     portal_url = fields.Char(
         'Portal Access URL', compute='_compute_portal_url',
         help='Customer Portal URL')
+    shared = fields.Boolean('Check shared Document', compute="_compute_share")
```

Comment:
Shouldn't it be `_compute_shareD`? We already have a `share` (without `d`) field on `res.users` which has a method `_compute_share`. The confusion seems easy.

Reply:
@mart-e , done :+1: